acai-ios/BowlOrderDetailViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  BowlOrderDetailViewController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+protocol DidSelectOption: class {
+    func deselectOption(at index: Int)
+    func selectOption(at index: Int, type: OrderCustomizationOptionType)
+}
+
+class BowlOrderDetailViewController: UIViewController {
+    
+    // MARK: View vars
+    var collectionView: UICollectionView!
+    var listAdapter: ListAdapter!
+    
+    // MARK: Data
+    var baseOptions: [OrderCustomizationOption]!
+    var bowlItem: BowlItem!
+    var toppingOptions: [OrderCustomizationOption]!
+    
+    // MARK: View Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        
+        // hard code
+        bowlItem = Acai.testBowl
+        toppingOptions = bowlItem.toppingOptions
+        baseOptions = bowlItem.baseOptions
+
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.backgroundColor = .white
+        collectionView.showsVerticalScrollIndicator = false
+        view.addSubview(collectionView)
+        
+        collectionView.snp.makeConstraints { (make) in
```

Comment:
@kevinchan159 are parentheses required here?

Reply:
oh yeah you should remove parentheses. i made a comment for that above